qa/L0_backend_python/io/test.sh

```diff
@@ -45,44 +45,62 @@ rm -fr *.log ./models
 pip3 uninstall -y torch
 pip3 install torch==1.9.0+cu111 -f https://download.pytorch.org/whl/torch_stable.html
 
-for i in {1..3}; do
-    model_name=dlpack_io_identity_$i
-    mkdir -p models/$model_name/1/
-    cp ../../python_models/dlpack_io_identity/model.py ./models/$model_name/1/
-    cp ../../python_models/dlpack_io_identity/config.pbtxt ./models/$model_name/
-    (cd models/$model_name && \
-              sed -i "s/^name:.*/name: \"$model_name\"/" config.pbtxt)
-done
+TRIALS="default decoupled"
 
-mkdir -p models/ensemble_io/1/
-cp ../../python_models/ensemble_io/config.pbtxt ./models/ensemble_io
+for trial in $TRIALS; do
+    export TRIAL=$trial
+    rm -rf ./models
 
-run_server
-if [ "$SERVER_PID" == "0" ]; then
-    echo -e "\n***\n*** Failed to start $SERVER\n***"
-    cat $SERVER_LOG
-    RET=1
-fi
+    if [ $trial = "default" ]; then
+        for i in {1..3}; do
+            model_name=dlpack_io_identity_$i
+            mkdir -p models/$model_name/1/
+            cp ../../python_models/dlpack_io_identity/model.py ./models/$model_name/1/
+            cp ../../python_models/dlpack_io_identity/config.pbtxt ./models/$model_name/
+            (cd models/$model_name && \
+                      sed -i "s/^name:.*/name: \"$model_name\"/" config.pbtxt)
+        done
+    else
+        for i in {1..3}; do
+            model_name=dlpack_io_identity_$i
+            mkdir -p models/$model_name/1/
+            cp ../../python_models/dlpack_io_identity_decoupled/model.py ./models/$model_name/1/
```

Comment:
Can you modify the dlpack_io_identity_decoupled model to actually send multiple responses back to the client? 
The number of responses can be determined by an additional input to the model.
I wonder how Triton will scale.

Reply:
> Can you modify the dlpack_io_identity_decoupled model to actually send multiple responses back to the client?
Can you elaborate on what we are testing by sending multiple responses? I think we are already testing sending multiple responses in `L0_decoupled`. Do you want to test sending multiple GPU responses back?

Reply:
Exactly. It will help us in checking if the multiple responses are returned properly in GPU.

Reply:
Added